test/unit/vaadin-progress-bar.test.ts

```diff
@@ -0,0 +1,158 @@
+import { expect, fixture, html } from '@vaadin/vaadin-component-dev-dependencies/testing.js';
+import { VaadinProgressBar } from '../../src/vaadin-progress-bar';
+
+describe('progress-bar', () => {
+  let progress: VaadinProgressBar;
+  let value: HTMLElement;
+  let tagName: string;
+
+  beforeEach(async () => {
+    progress = await fixture(
+      html`
+        <vaadin-progress-bar></vaadin-progress-bar>
+      `
+    );
+    value = progress.renderRoot.querySelector('[part="value"]') as HTMLElement;
+    tagName = progress.tagName.toLowerCase();
+  });
+
+  describe('custom element definition', () => {
+    it('should be defined in custom element registry', () => {
+      expect(customElements.get(tagName)).to.be.ok;
+      expect(progress instanceof VaadinProgressBar).to.be.ok;
+    });
+
+    it('should have a valid static "is" getter', () => {
+      expect(customElements.get(tagName).is).to.equal(tagName);
+    });
+
+    it('should have a valid version number', () => {
+      expect(customElements.get(tagName).version).to.match(/^(\d+\.)?(\d+\.)?(\d+)(-(alpha|beta)\d+)?$/);
+    });
+  });
+
+  function getProgressValue(element: VaadinProgressBar) {
+    const prop = '--vaadin-progress-value';
+    window.getComputedStyle(element).getPropertyValue(prop);
```

Comment:
```suggestion
    return window.getComputedStyle(element).getPropertyValue(prop);
```

Please add `return` statement. Tests are failing because of this line now.

Reply:
done